### ISSUE TITLE
Irregular tagging of releases

### ISSUE THREAD

**Issue by onlyjob** [mozilla/lightbeam-we#248]
Not all releases in https://addons.mozilla.org/en-US/firefox/addon/lightbeam/versions/ are tagged in this repository (e.g. tag `2.1.0` is missing).

Perhaps this is something to improve in release management...

**Comment by princiya** [comment]
Yes sure, I have some timewise bandwidth coming from my end from September, and I hope to get back to working on Lightbeam.
